### ISSUE TITLE
Difference between DP and DL (Google Analytics Debugger)

### ISSUE THREAD

**Issue by khryst** [react-ga/react-ga#209]
Hi everyone,

I installed the react-ga on my site and when I try to check the data in Google Analytics Debugger I ses that DP (document protocol) and dl (document location) are sometimes different :

https://user-images.githubusercontent.com/33697367/34565174-0b6fbfb8-f15a-11e7-8d4f-dbc4650836b6.png

It has to be like this  dl : mysite.com/example and dp : /example

And not dl : mysite.com/example and dp : /

Basically, the full URL and page path are different.

I am afraid that it can has an impact on the data.

Vahe you already met this kind of problem ?

Thanks

**Comment by SimeonC** [comment]
I think this is the same issue as in bug #210 (I just read that one first). There's a workaround in that issue, can you confirm it's the same?
